devo/sender/data.py

```diff
@@ -49,7 +57,7 @@ class SenderConfigSSL:
     """
     def __init__(self, address=None, key=None, cert=None, chain=None,
                  pkcs=None, sec_level=None, check_hostname=True,
-                 verify_mode=None):
+                 verify_config=False, verify_mode=None):
```

Comment:
Having the default behaviour set as true was my original idea but it breaks most tests in tests/sender and a couple in other places, the problem is that the certificates in the test folder are not compatible with localhost (since localhost has no certificate on it's own so trying to validate if a particular certificate is compatible with it will fail).

During the development I though a few solutions to this, `False` by default, mock `check_config_certificate_address` to return `True` in all other test cases that uses the Sender object, modify the method `check_config_certificate_address` to ignore a verification against localhost and simply return `True`.

Reply:
I finally think it is better to leave it as `False` since, otherwise, we would be introducing breaking changes.